package.json

```diff
@@ -80,21 +80,22 @@
     "url": "https://github.com/videojs/video.js.git"
   },
   "dependencies": {
-    "@babel/runtime": "^7.4.5",
+    "@babel/runtime": "^7.5.5",
     "@videojs/http-streaming": "1.10.3",
     "global": "4.3.2",
     "keycode": "^2.2.0",
     "safe-json-parse": "4.0.0",
     "tsml": "1.0.1",
     "videojs-font": "3.2.0",
     "videojs-vtt.js": "^0.14.1",
-    "xhr": "2.4.0"
+    "xhr": "github:brandonocasey/xhr#perf/less-external"
   },
   "devDependencies": {
     "@babel/cli": "^7.4.4",
     "@babel/core": "^7.4.5",
     "@babel/node": "^7.4.5",
-    "@babel/plugin-transform-runtime": "^7.4.4",
+    "@babel/plugin-external-helpers": "^7.2.0",
```

Comment:
Last time I played with it, using the external helpers actually made the file size much bigger, so, we should be careful with it.

Reply:
I will do some file size tests without it, and just the xhr change.